test/integration/mixer.yaml.tmpl

```diff
@@ -8,8 +69,14 @@ metadata:
     app: mixer
 spec:
   ports:
-  - port: 9091
-    name: tcp
+  - name: tcp
+    protocol: TCP
+    port: 9091
+    targetPort: 9091
```

Comment:
nit, you don't have to specify targetPort if it's same as port. TCP port is also default. Saves some keystrokes.